store/httpstore.go

```diff
@@ -92,8 +93,38 @@ func (s HTTPStore) SetMeta(name string, blob json.RawMessage) error {
 	return err
 }
 
+func (s HTTPStore) SetMultiMeta(metas map[string]json.RawMessage) error {
```

Comment:
Is there some better type that could be used here? There must be something better that we could do than string->raw json for pushing updates.